Dockerfile

```
@@ -36,21 +37,21 @@
 
       # Environment settings
       ENV PENTAHO_JAVA_HOME=$JAVA_HOME
-      ENV PENTAHO_HOME=$HOME/pentaho
       ENV CARTE_PORT=8080
-      ENV KETTLE_HOME=${PENTAHO_HOME}/data-integration
+      ENV KETTLE_HOME=${HOME}/data-integration
       ENV PATH=${KETTLE_HOME}:$PATH
 
-      # Create PENTAHO_HOME directory
-      RUN mkdir ${PENTAHO_HOME}
-      RUN chmod 775 ${PENTAHO_HOME}
+      # Create PENTAHO_HOME directory and make it writable
+      RUN mkdir ${KETTLE_HOME}
+      RUN chmod 775 ${KETTLE_HOME}
```

eigentlich kann man das auch zusammenfassen
```suggestion
      RUN mkdir ${KETTLE_HOME} && chmod 775 ${KETTLE_HOME}
```